frontend/pages/404.tsx

```diff
@@ -0,0 +1,22 @@
+import Layout from "@components/templates/Layout";
+import React from "react";
+import { Heading, Text } from "@chakra-ui/react";
+import Image from "next/image";
+
+const NotFound404 = () => {
```

Comment:
![bitmoji](https://sdk.bitmoji.com/render/panel/0d656257-fc44-4ef1-b9af-715a09b6afdf-cbc08238-ab4d-4b30-bbee-619cefc93217-v1.png?transparent=1&palette=1&width=246)

```suggestion
import Image from "next/image";
import { NextPage } from "next";

const NotFound404: NextPage = () => {
```